### PR TITLE
feat: add updatesService

### PR DESCRIPTION
**Pull request by thepassle**
closes https://github.com/Code-for-All/lockdown/issues/79
closes https://github.com/Code-for-All/lockdown/issues/143

### CODE REVIEW

File: data/updates.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (ilyasfoo) [inline data/updates.json:1]: This file will conflict with script generated one https://github.com/Code-for-All/lockdown/blob/2fe97699157b78d330cf373dc33102655ec880d5/data/updates.json

Comment (thepassle) [reply data/updates.json:1]: How will this conflict? It looks similar? Is it just the fact that a `link` is not always available?

Comment (ilyasfoo) [reply data/updates.json:1]: I meant this PR will conflict because my PR also contains `updates.json`

Comment (thepassle) [reply data/updates.json:1]: Ah yes, you're right. No worries, I'll fix that on my end, mine was a placeholder until yours was done 🙂

**Pull request merged**